server/logger.js

```diff
@@ -1,7 +1,18 @@
 const log = require("pino")();
 const expressPino = require("express-pino-logger")({ level: "silent", logger: log });
 
+const auditLogger = log.child({ tag: "AUDIT" });
+
+const auditLogEnabled = process.env.AUDIT_LOGGING !== "false" && process.env.AUDIT_LOGGING !== false;
```

Comment:
I would move this to https://github.com/aerogear/data-sync-ui/blob/master/server/config.js and import it from there so that we have all env vars in one file.

Reply:
Addressed this here: https://github.com/aerogear/data-sync-ui/pull/89